File: python/practices/time.py
time = int(input("what hour is it in military time? "))
if time == 24:
    print("GO TO SLEEP IT'S MIDNIGHT!")
elif time == 1:
    print("it's way to early to be awake")
elif time < 0:
    if time >= -24:
        print("that's yesterday")
    else:
        print("I'm not a time machine")
elif time == 12:
    print("It's noon")
elif time == 16:
    print("It's time to go home")
elif time == 0:
    print("ZZZZZZZZZZZZZZZZZZZZZZZZZ")
elif time > 24:
    if time <=48:
        print("that's tomarrow")
    else:
        print("I'm not a time machine, come back later and ask me again")
elif time < 5:
    print("go back to sleep")
elif time < 6:
    print("you're up early")
elif time < 12:
    print("Good morning")
elif time < 17:
    print("Good afternoon")
elif time < 21:
    print("Good evening")
elif time < 23:
    print("Good Night")
elif time < 24:
    print("Go To Bed")
else:
    print("WITCHCRAFT!")

File: C/practices/time.c
#include <stdio.h>
#include <stdbool.h>


int main(){
    int time;

    printf("what hour is it in military time? ");
    scanf("%d",&time);
    if (time == 0){
        printf("GO TO SLEEP IT'S MIDNIGHT!");
    }
    else if (time == 24){
        printf("Sorry, but thats not how military time works, for midnight put in 0");
    }
    else if (time == 2){
        printf("it's way to early to be awake");
    }
    else if (time < 0){
        if (time >= -24){
            printf("that's yesterday");
        }
        else{
            printf("I'm not a time machine");
        }
    }
    else if (time == 12){
        printf("It's noon");
    }
    else if (time == 16){
        printf("It's time to go home");
    }
    else if (time == 1){
        printf("ZZZZZZZZZZZZZZZZZZZZZZZZZ");
    }
    else if (time > 24){
        if (time <=48){
            printf("that's tomarrow");
        }
        else{
            printf("I'm not a time machine, come back later and ask me again");
        }
    }
    else if (time < 5){
        printf("go back to sleep");
    }
    else if (time < 6){
        printf("you're up early");
    }
    else if (time < 12){
        printf("Good morning");
    }
    else if (time < 17){
        printf("Good afternoon");
    }
    else if (time < 21){
        printf("Good evening");
    }
    else if (time < 23){
        printf("Good Night");
    }
    else if (time < 24){
        printf("Go To Bed");
    }
    else{
        printf("WITCHCRAFT!");
    }


    return(0);
}